ansible/roles/windows/common/tasks/main.yml

```diff
@@ -11,3 +11,28 @@
 - name: Turn off firewall
   win_shell: |
     Get-NetFirewallProfile | Set-NetFirewallProfile -Enabled False
+
+- name: Install cygwin
+  win_chocolatey:
+    name: cyg-get
+    state: present
+
+- name: Check if sshd service installed
+  win_service:
+    name: sshd
+  register: sshd_service
+
+- name: Install OpenSSH
+  win_shell: cyg-get openssh
+  when: not sshd_service.exists
+
+- name: Create passwd file
+  win_shell: "bin/mkpasswd > etc/passwd"
+  args:
+    chdir: "c:/tools/cygwin"
+    creates: "c:/tools/cygwin/etc/passwd"
```

Comment:
Could you define "c:/tools/cygwin"  as a variable as it was in original playbook? This will make it easier to change it in case the default directory changes.